qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -315,6 +317,11 @@ class BaseSpecification extends Specification {
 
         BaseService.useBasicAuth()
         BaseService.setUseClientCert(false)
+        //TODO(janisz): figure out why Sensor is unhealthy at the end of UpgradesTest
```

Comment:
Is there a JIRA ticket we can link in the TODO here ?
Do we want to check only on that `UpgradesTest` or on any test instance ?